src/components/Tree/Tree.module.css

```diff
@@ -0,0 +1,62 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+.tree {
+    max-width: 800px;
+    text-align: left;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+    font-family: var(--f-font-standard);
+    font-weight: 400;
```

Comment:
isn't there a var for `font-weight`s?

Reply:
not yet

Reply:
not sure if we need/want to add this to the css-vars? wdyt @SamuelAlev / @deniaz ?